src/Components/ModuleForm/ModuleForm.css

```diff
@@ -0,0 +1,128 @@
+.button-enabled {
+  color: white;
+  border: none;
+  border-radius: 10px;
+  padding: 1%;
+  height: 2%;
+  margin-left: 5%;
+  cursor: pointer;
+}
+
+/*USE FE LOGIC TO DISABLE BUTTON*/
+.button-disabled {
+  background-color: #ef4444;
+  border: none;
+  border-radius: 5px;
+  padding: 10px;
+  cursor: not-allowed;
+}
+
+.module-form {
+  background-color: rgb(255, 255, 255);
+  display: flex;
+  flex-direction: row;
+  padding: 3% 3% 2% 2%;
+  width: 100%;
+  justify-content: center;
+}
+
+@media screen and (max-width: 600px) {
+  .module-form {
+    flex-direction: column;
+  }
+}
+
+.module-list-title {
+  width: 100%;
+  align-items: left;
+}
+/* Thead */
+/* TO FIX ALL OF THE FOLLOWING*/
```

Comment:
Such comments are kinda redunant :) Mostly we will not do it, and therefore can just not keep this comment but rather keep a task in your todo list